docs/src/app/(public)/(content)/react/overview/releases/page.mdx

```diff
@@ -3,6 +3,244 @@
 <Subtitle>Changelogs for each Base UI release.</Subtitle>
 <Meta name="description" content="Changelogs for each Base UI release." />
 
+## v1.0.0-beta.5
+
+**Nov 13, 2025**
+
+### Accordion
+
+- **Breaking change:** Change `multiple` prop to be false by default and add a demo ([#3141](https://github.com/mui/base-ui/pull/3141))
```

Comment:
Some BCs didn't seem like they need further explanation. 🤔 
Feel free to suggest one if you feel otherwise.